src/where-did-it-come-from.md

```diff
@@ -0,0 +1,37 @@
+# Where did it come from?
+
+---
+
+## A Little Bit of History
+
+* Rust began around 2008
+* An experimental project by Graydon Hoare
+* Adopted by Mozilla
+* Presented to the general public as version 0.4 in 2012
+
+---
+
+## Focus
+
+* Rust lost many features from 2012 to 2014
+* Garbage collector, evented runtime, complex error handling
+* All present once, now gone
```

Comment:
Redundant information

Reply:
I think it's still useful to show that Rust once had "more" and people modified it to have "less", as opposed to Rust only having "less" because people haven't gotten around to adding "more".

Reply:
Unless you mean the bullet point itself, in which case, yeah I can buy that.